src/store/getters/state-getters.ts

```diff
@@ -256,7 +265,7 @@ export const isFilingValid = (state: any): boolean => {
 
 export const getDefaultCorrectionDetailComment = (state: any): string => {
   const filingDate = state.stateModel.originalIA.header.date.split('T')[0] || ''
-  return `Correction for Incorporation Application. Filed on ${filingDate}`
+  return `Correction for Incorporation Application filed on ${filingDate}`
```

Comment:
I know this wasn't specced, but what do you think about a period at the end?

Reply:
Updated!